packages/core/test/input.test.ts

```diff
@@ -0,0 +1,106 @@
+import { assign, interpret } from '../src';
+import { createMachine } from '../src/Machine';
+
+describe('input', () => {
```

Comment:
should we also support `input` in other actor types? `fromPromise` & co could also accept this

Reply:
Yes

Reply:
Supported everywhere!